src/main/java/com/example/bytex/user/UserEntityController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.bytex.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "users")
+public class UserEntityController {
+
+    UserEntityService userEntityService;
+
+    @Autowired
+    public UserEntityController(UserEntityService userEntityService) {
+        this.userEntityService = userEntityService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<UserEntity>> getUsers(){
+        List<UserEntity> allUsers = userEntityService.getUsers();
+
+        if(allUsers.size() != 0)
+        {
+            return new ResponseEntity<>(allUsers, HttpStatus.OK);
+        }
+
+        throw new ResponseStatusException(HttpStatus.NO_CONTENT, "Nothing to show");
+
+
+    }
+
+    @PostMapping
+    public ResponseEntity<List<UserEntity>> addNewUser(@RequestBody UserEntity userEntity)
+    {
+        userEntityService.addNewUser(userEntity);
+        return new ResponseEntity<>(userEntityService.getUsers(), HttpStatus.OK);
```

Comment:
When creating a resource, the response should be the newly created resource. In this case, you should return the saved `userEntity`. This will contain all the original fields (got from request initiator) as well as the ID (which is generated by backend/DB).  For more information regarding RESTful apis, I recommend this article: https://martinfowler.com/articles/richardsonMaturityModel.html